typedoc.json

```diff
@@ -1,13 +1,12 @@
 {
   "$schema": "https://typedoc.org/schema.json",
-  "entryPoints": ["./src/HypercertExchangeClient.ts", "./src/types.ts"],
+  "entryPoints": ["./src/HypercertExchangeClient.ts", "./src/types.ts", "./guides/README.md"],
   "out": "./doc",
   "githubPages": false,
   "readme": "none",
   "gitRemote": "https://github.com/hypercerts-org/marketplace-sdk",
   "disableSources": true,
   "cleanOutputDir": false,
 
-  "hideBreadcrumbs": true,
-  "hideInPageTOC": true
+  "projectDocuments": ["./guides/readme.md"],
```

Comment:
This one should also be all caps like above, shouldn't it?

Reply:
Addressed